src/bebone/core/types/utils/trait_bryan_angles.h

```diff
@@ -0,0 +1,158 @@
+                    0.0f,               0.0f,    0.0f,                  1.0f
+                }};
+        #endif
+    }
+
+    static inline Matrix<f32, 4, 4> trait_bryan_angle_yzx(const Vec3f& rotation) {
+        const f32 c3 = std::cos(rotation.z);
+        const f32 s3 = std::sin(rotation.z);
+        const f32 c2 = std::cos(rotation.x);
+        const f32 s2 = std::sin(rotation.x);
+        const f32 c1 = std::cos(rotation.y);
+        const f32 s1 = std::sin(rotation.y);
+
+        #ifndef BEBONE_TYPES_MATRIX_COLLUM_MAJOR_ORDER
+                return {{
+                    c1*c2,      s1*s3 - c1*c3*c2,   c3*s1 + c1*s2*s3,   0.0f,
+                    s2,         c2*c3,               -c2*s3,             0.0f,
+                    -c2*s1,     c1*s3 + c3*s1*s2,   c1*c3 - s1*s2*s3,   0.0f,
+                    0.0f,       0.0f,               0.0f,               1.0f
+                }};
+        #else
+                return {{
+                        c1*c2,              s2,         -c2*s1,             0.0f,
+                        s1*s3 - c1*c3*c2,   c2*c3,       c1*s3 + c3*s1*s2,   0.0f,
+                        c3*s1 + c1*s2*s3,   -c2*s3,     c1*c3 - s1*s2*s3,   0.0f,
+                        0.0f,       0.0f,               0.0f,               1.0f
+                }};
+        #endif
+    }
+
+    static inline Matrix<f32, 4, 4> trait_bryan_angle_zyx(const Vec3f& rotation) {
+        const f32 c3 = std::cos(rotation.z);
+        const f32 s3 = std::sin(rotation.z);
+        const f32 c2 = std::cos(rotation.x);
+        const f32 s2 = std::sin(rotation.x);
+        const f32 c1 = std::cos(rotation.y);
+        const f32 s1 = std::sin(rotation.y);
+
+        #ifndef BEBONE_TYPES_MATRIX_COLLUM_MAJOR_ORDER
+                return {{
+                    c1*c2,      c1*s2*s3 - c3*s1,   s1*s3 + c1*c3*s2,   0.0f,
+                    c2*s1,      c1*c3 + s1*s2*s3,   c3*s1*s2 - c1*s3,   0.0f,
+                    -s2,        c2*s3,              c2*c3,              0.0f,
+                    0.0f,       0.0f,               0.0f,               1.0f
+                }};
+        #else
+                return {{
+                        c1*c2,              c2*s1,              -s2,        0.0f,
+                        c1*s2*s3 - c3*s1,   c1*c3 + s1*s2*s3,   c2*s3,      0.0f,
+                        s1*s3 + c1*c3*s2,   c3*s1*s2 - c1*s3,   c2*c3,      0.0f,
+                        0.0f,               0.0f,               0.0f,       1.0f
+                }};
+        #endif
+    }
+
+    static inline Matrix<f32, 4, 4> trait_bryan_angle_zxy(const Vec3f& rotation) {
+        const f32 c3 = std::cos(rotation.z);
+        const f32 s3 = std::sin(rotation.z);
+        const f32 c2 = std::cos(rotation.x);
+        const f32 s2 = std::sin(rotation.x);
+        const f32 c1 = std::cos(rotation.y);
+        const f32 s1 = std::sin(rotation.y);
+
+        #ifndef BEBONE_TYPES_MATRIX_COLLUM_MAJOR_ORDER
```

Comment:
Grammar error
Change `BEBONE_TYPES_MATRIX_COLLUM_MAJOR_ORDER` to `BEBONE_TYPES_MATRIX_COLUMN_MAJOR_ORDER`